README.md

```diff
@@ -161,7 +161,7 @@ Note: if you want to train and evaluate on a task, that task must be in both ``t
 
 To add new tasks, you should:
 
-1. Add your data to the ``data_dir`` you intend to use. When constructing your task class (see next bullet), make sure you specify the correct subfolder containing your data. 
+1. Add your data to the ``data_dir`` you intend to use. When constructing your task class (see next bullet), make sure you specify the correct subfolder containing your data. It is recommended that you shuffle your training and validation data if there are many rows (>10k) to avoid training artefacts. Indeed, the framework loads 10k examples at a time in memory then shuffles those. E.g: in a binary classification setting, if your data is sorted by label, the model would go through all the positive examples before finding a negative, so would learn to always predict 1. If you reach 100% accuracy in training before one epoch, this is likely the case.
```

Comment:
Might be better to add this note (sentences 2:end) as a separate paragraph for emphasis?

Reply:
Also maybe add a TODO to the code to have a better solution to this, even something as simple as running `shuf` on the instance-record file emitted by the indexing step (this should work since each line is one b64-encoded example).